Branch2018/test1/Program.cs

```diff
@@ -0,0 +1,195 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace test1
+{
+    class Program
+    {
+        //public List<Venta> ArraynuevaVenta = new List<Venta>();
+        //public List<Cliente> ArrayclienteNuevo = new List<Cliente>();
+        static void Main(string[] args)
+        {
+            
+             int _OpcionMenu = 0;            
+            while(_OpcionMenu!=6){
+                Console.WriteLine("--------------------");
+                Console.WriteLine("Menu de Facturacion");
+                Console.WriteLine("--------------------");
+                Console.WriteLine("1. Registro de Ventas");
+                Console.WriteLine("2. Registro de Clientes");
+                Console.WriteLine("3. Registro de vendedores");
+                Console.WriteLine("4. Registro de Productos");
+                Console.WriteLine("5. Reportes");
+                Console.WriteLine("6. Reportes");
+                Console.WriteLine("Seleccione una opcion");
+                _OpcionMenu = int.Parse(Console.ReadLine());
+                        
+                switch(_OpcionMenu){                     
+                    case 1: registroVentas();break;
+                    case 2: registroClientes();break;                
+                    case 3:registroVendedores();break;
+                    case 4:registroProductos();break;
+                    case 5:menuReporte();break;
+                    default : Console.WriteLine("Gracias por su visita");break;
+                }
+            }
+        }
+
+        public static void menuReporte(){

```

Comment:
names of methods must to start with uppercase(Pascal case).

Reply:
ready